runtime/lua/vim/lsp/buf.lua

```diff
@@ -1062,7 +1061,7 @@ function M.execute_command(command_params)
     arguments = command_params.arguments,
     workDoneToken = command_params.workDoneToken,
   }
-  request(ms.workspace_executeCommand, command_params)
+  lsp.buf_request(0, ms.workspace_executeCommand, command_params)
```

Comment:
Also for potential follow up: We should deprecate `execute_command` because commands are always specific to a single server.  We could make `Client:_exec_cmd` public instead.

Reply:
#30932